test/scripts/prepare_agent_config.sh

```diff
@@ -1,24 +1,6 @@
 #!/usr/bin/env bash
 
-SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
-IP=$("${SCRIPT_DIR}"/get_ext_ip.sh)
-OUTFILE=bin/agent/etc/flightctl/config.yaml
-mkdir -p bin/agent/etc/flightctl/certs
 cp ~/.flightctl/certs/ca.crt bin/agent/etc/flightctl/certs/ca.crt
 cp ~/.flightctl/certs/client-enrollment.{crt,key} bin/agent/etc/flightctl/certs/
```

Comment:
```suggestion
```

The agent config has the CA embedded and the enrollment cert and key as well, so we don't need to copy that anymore I believe